src/collective/lazysizes/upgrades/v2/configure.zcml

```diff
@@ -25,6 +25,13 @@
         handler="..cook_javascript_resources"
         />
 
+    <genericsetup:upgradeDepends
```

Comment:
I think this just happen when the upgrade step has the same from/to number (in this case from 2 to 3)